### PR TITLE
Dev/3.0

### PR DESCRIPTION
**Pull request by jrobinAV**
The last dev/3.0 branch to remove...

### CODE REVIEW

File: docs/manuals/core/concepts/scope.md [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
-- `Scope.PIPELINE`
 - `Scope.SCENARIO` (Default value)
 - `Scope.CYCLE`
 - `Scope.GLOBAL`
 
 Each data node has a scope. It is an attribute provided by the `DataNodeConfig^` and represents the _visibility_ of
 the data node in the graph of entities. The entities can be described as a graph where each node belongs to one or
-several upper nodes. A data node 'belongs' to at least one pipeline, which belongs to at least one scenario, which
-belongs to a single cycle.
+several upper nodes. A data node 'belongs' to at least one scenario, which belongs to a single cycle.
```
Comment (FabienLelaquais) [inline docs/manuals/core/concepts/scope.md]: 'upper' is a bit unfortunate here... May be 'parent'?

File: docs/relnotes.md [hunk 2]
```diff
@@ -12,7 +12,19 @@ This is the list of changes to Taipy releases as they were published.
...
+
+Not published yet.
+
+### New Features
+
+### Improvements and changes
+
+<h6 style="font-size: 1.2em"><strong><code>taipy-core</code></strong></h6>
+
+- `Scope.PIPELINE` has been removed as a value of `Scope^`.
```
Comment (jrobinAV) [inline docs/relnotes.md]:
```suggestion
- `Scope.PIPELINE` has been removed from `Scope^` values.
```

**Review by FabienLelaquais — APPROVED** [review]
Tiny comment...

File: docs/relnotes.md [hunk 3]
```diff
@@ -12,7 +12,19 @@ This is the list of changes to Taipy releases as they were published.
...
+## Community edition: 3.0 (Work in progress)
+
+Not published yet.
+
+### New Features
+
+### Improvements and changes
+
+<h6 style="font-size: 1.2em"><strong><code>taipy-core</code></strong></h6>
+
```
Comment (jrobinAV) [inline docs/relnotes.md:25]:
```suggestion
3.0.0

```

File: docs/manuals/core/concepts/scope.md (hunk 1 above)
Comment (jrobinAV) [inline docs/manuals/core/concepts/scope.md]:
```suggestion
several parent nodes. A data node 'belongs' to at least one scenario belonging to a single cycle.
```

**Pull request merged**